packages/esm-patient-forms-app/src/forms/forms-context.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { createContext, useContext, useState, useMemo } from 'react';
+
+interface FormsContextState {
+  pageSize: number;
+  currentPage: number;
+  searchTerm: string;
+  setPageSize: (size: number) => void;
+  setCurrentPage: (page: number) => void;
+  setSearchTerm: (term: string) => void;
+}
+
+const FormsContext = createContext<FormsContextState | undefined>(undefined);
+
+interface FormsProviderProps {
+  children: React.ReactNode;
+  defaultPageSize?: number;
+  defaultCurrentPage?: number;
+}
+
+export const FormsProvider: React.FC<FormsProviderProps> = ({
```

Comment:
I don't really understand the utility of having this context setup? It doesn't seem to be used anywhere that mounts two components that need to share state, it's always using the default values, so it's not like we need to move pagination state captured at a higher-level component to a lower-level component?